test/oCookieMessage.test.js

```diff
@@ -51,6 +57,8 @@ describe("Cookie Message", () => {
 	});
 
 	describe('HTML', () => {
+		before(waitForPageshowEvent);
```

Comment:
Nice 👍

Reply:
Unfortunately it times out during tests, I think because it's bound at the wrong point in the lifecycle

Reply:
Yeah it looks like it's fired when the tests load and all the tests run without reloading the page. 

🤔 

It might be better to have the `if (this.shouldShowCookieMessage()) { [..] }` logic outside of the event handler anyway? In case it the cookie message should be initialised after the `pageshow` event for some reason we still want to show the cookie message. That would have the existing tests pass. To test the new behaviour I wonder if it'd possible to mock `window.location.reload` and fire a false `pageshow` event? (we do a [similar thing to test clicks](https://github.com/Financial-Times/o-table/blob/7d0dd75461270a4f44baf8f318dc19712c3e09e0/test/BaseTable.test.js#L31))

Reply:
thinking about it, if the reader ignores the cookie message, and navigates back and fourth e.g. from the frontpage to articles, that could be a lot of reloads? 😬

Reply:
Agreed, which is why it was purely a fallback, and only intended to handle the case that `document.cookie` wasn't readable. 

Since that isn't the case we're able to rely on the existing CookieMessage methods 👍